cli/commands.js

```diff
@@ -108,6 +108,10 @@ const defaultCommands = {
         option: '--skip-user-management',
         desc: 'Skip the user management module during app generation',
       },
+      {
+        option: '--unidirectional-relationships',
```

Comment:
this doesn't look like an option, what about `--generate-unilateral-relationships` ?

Reply:
I don't see the need for `generate`.